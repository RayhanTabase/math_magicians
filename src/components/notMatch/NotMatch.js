const NotMatch = () => (
  <div>
    <h3>Page does not exist</h3>
  </div>
);
export default NotMatch;
