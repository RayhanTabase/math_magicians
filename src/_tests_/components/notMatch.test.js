import React from 'react';
import renderer from 'react-test-renderer';
import NotMatch from '../../components/notMatch/NotMatch'

it('renders correctly', () => {
  const tree = renderer
    .create(<NotMatch />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});



