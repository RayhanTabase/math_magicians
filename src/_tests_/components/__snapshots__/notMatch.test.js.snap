// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h3>
    Page does not exist
  </h3>
</div>
`;
